src/Tools/vc_viral_load.php

```diff
@@ -35,7 +36,7 @@
 }
 
 //copy genome to local data folder
-$parser->execTool("Tools/data_setup.php", "-build {$build_viral}");
+if (!$no_sync) $parser->execTool("Tools/data_setup.php", "-build {$build_viral}");
```

Comment:
warum müssen wir in dem Tool syncen?
Eigentlich sollte data_setup nur in den Pipelines aufgerufen werden.,,

Reply:
Nimm das einfach aus dem data_folder. Bei der Größe muss das nicht ins tmp